### PR TITLE
[ShaderGraph] [bugfix 1271598] Check for null when attempting to get vt property

### PR DESCRIPTION
**Pull request by cdxntchou**
### Purpose of this PR
Fixing bug: https://fogbugz.unity3d.com/f/cases/1271598/

When opening a graph containing a procedural virtual texture (PVT) node, but without the PROCEDURAL_VT_IN_GRAPH define in the project, the PVT node is replaced with a placeholder.  The SampleVirtualTexture node attempts to read data from the PVT node, but doesn't check that the PVT node returned a non-null property before doing so.

This change simply checks for null before using the returned property.

---
### Testing status

Tested against repro case in windows editor.
Tested with PROCEDURAL_VT_IN_GRAPH enabled and disabled
Loading existing graphs, creating new graphs using the node
Tested Undo/redo creation of PVT nodes and connection of wires from PVT to SVT nodes.

**Yamato**: (Select your branch):
https://yamato.prd.cds.internal.unity3d.com/jobs/902-Graphics

Any test projects to go with this to help reviewers?

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/SampleVirtualTextureNode.cs [hunk 1]
```diff
@@ -426,62 +426,68 @@ public void GenerateNodeFunction(FunctionRegistry registry, GenerationMode gener
...
         {
+            bool success = false;
             if (IsSlotConnected(VirtualTextureInputId))
             {
                 var vtProperty = GetSlotProperty(VirtualTextureInputId) as VirtualTextureShaderProperty;
-                int layerCount = vtProperty.value.layers.Count;
-
-                var layerOutputVariables = new List<string>();
-                for (int i = 0; i < layerCount; i++)
+                if (vtProperty != null)
```
Comment (cdxntchou) [inline com.unity.shadergraph/Editor/Data/Nodes/Input/Texture/SampleVirtualTextureNode.cs:433]: Github diff is getting confused by the added indent from the new { } block added by this if()..  there are no changes in the block below other than adding success= true.

**Comment by LandonTownsendUnity** [comment]
This PR is missing the changelog update.

**Review by LandonTownsendUnity — APPROVED** [review]
Developer manual testing is sufficient.

**Comment by cdxntchou** [comment]
Yamato all green except OSX metal jobs, which are failing on matching trunk version

**Pull request merged**